solr/core/src/test/org/apache/solr/search/TestExtendedDismaxParser.java

```diff
@@ -1794,6 +1798,37 @@ public void testOperatorsAndMultiWordSynonyms() throws Exception {
     );
   }
 
+  public void testOverlapTermScoringQueries() throws Exception {
```

Comment:
This new functionality should apply to the lucene QueryParser... wouldn't a test be better targeted there instead of edismax?

Reply:
It could go either place, I put it here based on following the work for adding autoGeneratePhraseQueries

Reply:
I see.  Nonetheless I think it belongs in TestSolrQueryParser.  I'd rather edismax tests stick to testing edismax and not LuceneQParser/SolrQueryParser stuff unless it's incidental.